### PR TITLE
Update hello Application to "6.0.0-beta.7"

### LINKED ISSUE #1858: Update dashboard plugin guide
### Description
A recent update in the dashboard plugin also exposes the registered endpoints from plugins. Update the content to describe how a plugin's endpoint can be invoked using the dashboard.

- Add information about how to enable the plugin using designated flags.
- Update the Hello application to the latest beta tag in `lisk-sdk-examples`.
- Invoke plugin endpoint using the dashboard

#### Affected branch / URL
`development`

#### Page location
`build-blockchain/using-dashboard.adoc

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Partially closes https://github.com/LiskHQ/lisk-docs/issues/1858

### Start blockchain client
`./bin/run start --config=config/custom_config.json
`
### Start front end
`npm start`

### CODE REVIEW

File: tutorials/hello/hello_client/config/default/config.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-		"dataPath": "/Users/mona/.lisk/beta-sdk-app",
+		"dataPath": "/Users/iamtalha/.lisk/beta-sdk-app",
 		"keepEventsForHeights": 300,
 		"logLevel": "info"
 	},
 	"rpc": {
-		"modes": ["ipc"],
+		"modes": [
+			"ipc"
+		],
```
Comment (Tschakki) [inline tutorials/hello/hello_client/config/default/config.json:10]: `allowedMethods` is missing here, without it the app wont start:

```
2023-08-22T09:22:39.506Z INFO Monas-MacBook-Pro.local application 52364 helloInfo Loading in-memory plugin
2023-08-22T09:22:39.514Z INFO Monas-MacBook-Pro.local engine 52364 [status=error err=Requested method or namespace is disabled in node config.] Failed to handle IPC request
    Error: Requested method or namespace is disabled in node config.
2023-08-22T09:22:39.522Z INFO Monas-MacBook-Pro.local application 52364 [errorCode=2 message=process.exit] Application shutdown started
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Left a small remark regarding config, rest LGTM 👍

File: tutorials/hello/hello_client/config/default/config.json (hunk 1 above)
Comment (TalhaMaliktz) [reply tutorials/hello/hello_client/config/default/config.json:10]: Please use `./bin/run start --config=config/custom_config.json` to start the app as suggested in the [README](https://github.com/LiskHQ/lisk-sdk-examples/tree/development/tutorials/hello/hello_client#start-a-node). The changes made in the PR are resultant of `lisk init` which means that by default the `allowedMethods` property is not present in the default config.

**Review by Tschakki — APPROVED** [review]
LGTM

**Review by Incede — APPROVED** [review]

**Pull request merged**